internal/cmd/completion.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func newCompletionsCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "completion",
+		Short: "Generate completions",
+		Long: fmt.Sprintf(`To load completions:
+
+Bash:
+
+  $ source <(%[1]s completion bash)
+
+  # To load completions for each session, execute once:
+  # Linux:
+  $ %[1]s completion bash > /etc/bash_completion.d/%[1]s
```

Comment:
I generally put my bash completion at  `~/.bash_complete/` , but I'm not sure how standard that is.

Someone trying to run this command as non-root will get an error, because they don't have permission to write to `/etc`.

Maybe this should use a directory under their home dir, or:

```
$ %[1]s completion bash | sudo tee /etc/bash_completion.d/%[1]s
```